packages/common-ui-web/src/components/VehicleDynamicWireframe/VehicleDynamicWireframe.tsx

```diff
@@ -0,0 +1,45 @@
+import { useMonkTheme } from '@monkvision/common';
+import { wireFrame } from '@monkvision/sights';
+import { PartSelectionOrientation, VehicleModel, VehiclePart } from '@monkvision/types';
+import { useMemo } from 'react';
+import { DynamicSVG } from '../DynamicSVG';
+import { usePartsOverlay } from './hooks';
+
+export interface VehicleDynamicWireframeProps {
+  vehicleModel: VehicleModel;
```

Comment:
This component should accept a `vehicleType` and map vehicle type to vehicle models